### PR TITLE
New reminders, Fixes #14 and fixes #9

### LINKED ISSUE #9: Reminder to water plants
@marykatefain

### LINKED ISSUE #14: Reminder to order groceries
How often does this happen again?

### PR DESCRIPTION
**Pull request by marykatefain**
Reminders to water plants daily and order groceries weekly
Changed questionable Twink language in weekly check-in

### CODE REVIEW

File: app/server.js [hunk 1]
```diff
@@ -95,6 +95,11 @@ twink.remind({ "hour": 19, "minute": 0 },
   "Time to do the dishes!"
 )
 
+//Watering Plants
```
Comment (alexgleason) [inline app/server.js]: Please put a space here between the `//` and `Watering`

File: app/server.js [hunk 2]
```diff
@@ -103,9 +108,14 @@ twink.remind({ "dayOfWeek": 1, "hour": 10, "minute": 0 },
   "Don't forget to take in the trash!"
 )
 
+//Ordering Groceries
+twink.remind({ "dayOfWeek": 5, "hour": 19, "minute": 0 },
```
Comment (alexgleason) [inline app/server.js]: Friday at 7pm? Not against it, but curious of your rationale.

File: app/server.js [hunk 3]
```diff
@@ -103,9 +108,14 @@ twink.remind({ "dayOfWeek": 1, "hour": 10, "minute": 0 },
...
+  "Don't forget to order groceries! Check the Wekan Grocery List for items \n\nhttps://board.uptwinkles.co/b/ffNduJZK6weDPaqDt/uptwinkles\n\n. If we need a lot, we should do a Fresh Grocer order. Otherwise, let's pick up produce this week. Who can take charge of doing this?"
+)
+
 // Weekly check-in
 twink.remind({ "dayOfWeek": 2, "hour": 20, "minute": 0 },
-  "Whaddup, fam‽ It's time for our weekly check-in. What is the progress on our kanban board?\n\nhttps://board.uptwinkles.co/b/ffNduJZK6weDPaqDt/uptwinkles\n\nI suggest meeting up if there's a lot to cover. Let's make sure everyone is assigned to something and the cards reflect our current state of being. Y'all are my favourite humans. Peace. ✌️"
+  "It's time for our weekly check-in! What is the progress on our kanban board?\n\nhttps://board.uptwinkles.co/b/ffNduJZK6weDPaqDt/uptwinkles\n\nI suggest meeting up if there's a lot to cover. Let's make sure everyone is assigned to something and the cards reflect our current state of being. Thanks!"
```
Comment (alexgleason) [inline app/server.js]: I won't block this change but I'm sad the interrobang was removed.

**Comment by alexgleason** [comment]
This is a test of your git skills. I like your commits and your commit messages so I'd prefer you keep them, but at least one commit should be altered before I merge. The blocker for me is the lack of a space in the comment.

File: app/server.js (hunk 1 above)
Comment (marykatefain) [inline app/server.js]: Is this literally just style nit-picking or does it matter in any real way?

**Comment by marykatefain** [comment]
*ahem* better?

**Pull request merged**